src/locale/id.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
+  'alpha': 'Input harus berupa huruf.',
```

Comment:
hi @albertstevelino , these validations is missing 
```fileType:$1
not-equal:$1
not-memberOf:$1
date
fqdn
imei
string

Reply:
thanks for the notes